apps/etterlatte-utbetaling/src/test/kotlin/utbetaling/TestHelper.kt

```diff
@@ -119,7 +119,7 @@ fun oppdragMedFeiletKvittering(
 fun utbetaling(
     id: UUID = UUID.randomUUID(),
     sakId: SakId = SakId(1),
-    sakType: Saktype = Saktype.BARNEPENSJON,
+    sakType: Saktype? = Saktype.BARNEPENSJON,
```

Comment:
Denne skjønte ikke jeg helt hvorfor må være optional? Ganske sikker på at man kan bruke denne funksjonen direkte for omstillingsstønad ved `utbetaling(sakType = SakType.OMSTILLINGSSTOENAD)`